server/login/views.py

```diff
@@ -127,6 +128,10 @@ def save_new_pin(email, user):
     request.data = {'password': first_part + second_part + third_part}
     # For e2e purposes, uncomment this line:
     # request.data = {'password': 'password'}
+    print('\n\n--------------------------')
+    print(request.data)
+    print('--------------------------\n\n')
```

Comment:
Are these required?
If not, please remove it

Reply:
no, we can remove that